product/dialogs/miq_dialogs/miq_provision_microsoft_dialogs_template.yaml

```diff
@@ -1,565 +0,0 @@
----
-:name: miq_provision_microsoft_dialogs_template
```

Comment:
We have to see what happens on seed - if these aren't removed we might need a data migration.

Reply:
They are deleted during seed:
```
>> MiqDialog.exists?(:name => "miq_provision_microsoft_dialogs_template")
=> true
$ rails db:seed
$ rails c
>> >> MiqDialog.exists?(:name => "miq_provision_microsoft_dialogs_template")
=> false
```